src/sonic-config-engine/config_samples.py

```diff
@@ -75,6 +75,39 @@ def generate_t1_sample_config(data):
         port_count += 1
     return data
 
+def generate_t1_smartswitch_sample_config(data):
+    data = generate_t1_sample_config(data)
+    data['DEVICE_METADATA']['localhost']['subtype'] = 'SmartSwitch'
+
+    bridge_name = 'bridge_midplane'
+    data['MID_PLANE_BRIDGE'] = {
+        'GLOBAL': {
+            'bridge': bridge_name,
+            'address': '169.254.200.254/24'
```

Comment:
does it make sense to have the 169.254.200.254 defined in a single place?

Reply:
done